### ISSUE TITLE
Support for Logic Pro X 

### ISSUE THREAD

**Issue by dherr0912** [bitfocus/companion-module-requests#341]
**PLEASE USE THE FOLLOWING PAGE IF YOU WANT SUPPORT FOR A NEW DEVICE OR SOFTWARE PROGRAM**: https://github.com/bitfocus/companion-module-requests/issues/new

I basically only need it to mute and unmute multiple channels for a livestream set at our church.

**Comment by josephdadams** [comment]
@dherr0912 please post a link to the Logic Pro X API that would allow Companion to control it over the network. My guess is that there isn’t one, and that the only real method of controlling it would he an AppleScript.

**Comment by dszoka** [comment]
Just an idea... Logic can be controller via midi ..

could we use 
https://github.com/bitfocus/companion-module-techministry-midirelay 

https://support.apple.com/en-au/guide/logicpro/lgcp0155c51f/mac

if I have time over the next couple weeks I might be able to test this

**Comment by josephdadams** [comment]
Yes. Just be sure to have the midi relay server program running on the Logic computer.

**Comment by dherr0912** [comment]
I found that yes I could have had midi relay running but volume control and plug-in control would have been a nightmare? Instead we got a 8 channel presonus faderport. Literally made for controlling the daw, no loose ends so far. The stream deck do and companion has been working great for controlling the other two computers and it’s programs. 

Damien Herrington

**Comment by Vermillionist** [comment]
Thank you all for your comments but I am having trouble following this. Is it possible to control Logic using the Stream Deck XL? if so, how?
